scapy/utils6.py

```diff
@@ -250,7 +250,7 @@ def in6_addrtovendor(addr):
     is returned on error, "UNKNOWN" if the vendor is unknown.
     """
     mac = in6_addrtomac(addr)
-    if mac is None:
+    if mac is None or conf.manufdb:
```

Comment:
I may be wrong, but shouldn’t it be `not conf.manufdb` ?

Reply:
You are right. I should not teach and fix at the same time ....